capture/genBitmaps.js

```diff
@@ -231,6 +231,7 @@ function processScenario (casper, scenario, scenarioOrVariantLabel, scenarioLabe
             selector: o,
             fileName: fileName,
             label: scenario.label,
+            isSameDimensions: scenario.isSameDimensions || config.isSameDimensions || config.defaultIsSameDimensions,
```

Comment:
do these need to be checking for falsly values? for example, if the scenario says false, but the config says true, the higher precedence scenario will be trumped